src/server/game/World/World.cpp

```diff
@@ -1328,7 +1328,11 @@ void World::LoadConfigSettings(bool reload)
     m_int_configs[CONFIG_AFK_PREVENT_LOGOUT] = sConfigMgr->GetIntDefault("PreventAFKLogout", 0);
 
     // Preload all grids of all non-instanced maps
-    m_bool_configs[CONFIG_PRELOAD_ALL_NON_INSTANCED_MAP_GRIDS] = sConfigMgr->GetIntDefault("PreloadAllNonInstancedMapGrids", false);
+    m_bool_configs[CONFIG_PRELOAD_ALL_NON_INSTANCED_MAP_GRIDS] = sConfigMgr->GetBoolDefault("PreloadAllNonInstancedMapGrids", false);
```

Comment:
fix for https://github.com/azerothcore/azerothcore-wotlk/commit/ab55aa4e5a3223ada9a0593dd4cbcb7a37fc5bc4?

Reply:
Not exactly a fix, as the parameter works, but it should be handled as boolean. Just wanted to include this here, I can also create a separate PR to fix this, but this is unnecessary in my opinion.

Reply:
I don't think we need a new one either.

Reply:
But I forgot to mention it in the PR description, will add this now.